content/blog/2020-04-dplyr-1-0-0-rowwise/index.Rmarkdown

```diff
@@ -0,0 +1,205 @@
+---
+title: 'dplyr 1.0.0: working within rows'
+author: Hadley Wickham
+date: '2020-04-10'
+slug: dplyr-1-0-0-rowwise
+categories:
+  - package
+tags:
+  - dplyr
+photo:
+  author: Oleksandr Hrebelnyk
+  url: https://unsplash.com/photos/ckZU2xZUjO8
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+This post is the latest in a series of post leading up the the dplyr 1.0.0 release. So far, the series has covered:
+
+* [Major lifecycle changes](https://www.tidyverse.org/blog/2020/03/dplyr-1-0-0-is-coming-soon/).
+* [New `summarise()` features](https://www.tidyverse.org/blog/2020/03/dplyr-1-0-0-summarise/).
+* [`select()`, `rename()`, `relocate()`](https://www.tidyverse.org/blog/2020/03/dplyr-1-0-0-select-rename-relocate/).
+* [Working `across()` columns](https://www.tidyverse.org/blog/2020/04/dplyr-1-0-0-colwise/).
+
+Today, I wanted to talk a little bit about the renewed `rowwise()` function that makes it easy to perform operations "row-by-row". I'll show how you can use `rowwise()` to compute summaries "by row", talk about how `rowwise()` is a natural pairing with list-columns, and show a couple of use cases that I think are particularly elegant. You can learn more about all of these topics in  [`vignette("rowwise")`](https://dplyr.tidyverse.org/dev/articles/rowwise.html).
+
+### Getting the dev version
+
+If you're interested in living life on the edge (or trying out anything you see in this blog post), you can install the development version of dplyr with:
+
+```{r, eval = FALSE}
+devtools::install_github("tidyverse/dplyr")
+```
+
+Note that the development version won't become 1.0.0 until it's released, but it has all the same features.
+
+```{r setup}
+library(dplyr, warn.conflicts = FALSE)
+```
+
+## Basic operation
+
+`rowwise()` works like `group_by()` in the sense that it doesn't change what the data looks like; it changes how dplyr verbs operate on it. Let's see how this works with a simple example. Here I have some imaginary test results for students in a class:
```

Comment:
```suggestion
`rowwise()` works like `group_by()` in the sense that it doesn't change what the data looks like; it changes how dplyr verbs operate on the data. Let's see how this works with a simple example. Here I have some imaginary test results for students in a class:
```

Reply:
I don't love my wording change, but the double "it" in that last part was a tad confusing.